<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Google Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Spreadsheet Data Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/app_icon.png</normaloff>../icons/app_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
color: #212121;
}

QPushButton{
border-radius: 3px;
background-color: #f1f2f4;
padding: 5px;
border: 1px solid #cacaca;
box-shadow: 0 2px 5px 0 rgba(0,0,0,.26);
}

QLineEdit{
border-radius: 3px;
padding: 5px;
border: 1px solid #cacaca
}

QTableView::item{
padding: 7px;
font: 300 14px/18px Roboto,sans-serif;
}

QTableView::item:focus{
padding: 0;
}

QScrollBar{
background-color: #e5e5e5;
}

</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>127</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Google Sans</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="start_tab">
       <attribute name="title">
        <string>Start Page</string>
       </attribute>
       <attribute name="toolTip">
        <string>Welcome to Data Converter's</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_left_main_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Google Sans</family>
                <pointsize>26</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string>Spreadsheet Data Converter</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_left_sub_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Google Sans</family>
                <pointsize>17</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(20,146,255);</string>
              </property>
              <property name="text">
               <string>Get Started</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Google Sans</family>
                <pointsize>13</pointsize>
                <weight>50</weight>
                <italic>true</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Select a module below to open a spreadsheet</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="module_selection_layout" stretch="0,0">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item alignment="Qt::AlignRight|Qt::AlignVCenter">
                 <widget class="QPushButton" name="push_btn_fin_plate">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>180</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Google Sans</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Fin Plate</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
                 <widget class="QPushButton" name="push_btn_tension_member">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>180</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Google Sans</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Tension Member</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item alignment="Qt::AlignRight|Qt::AlignVCenter">
                 <widget class="QPushButton" name="push_btn_bc_end_plate">
                  <property name="minimumSize">
                   <size>
                    <width>180</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Google Sans</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>BC End Plate</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
                 <widget class="QPushButton" name="push_btn_cleat_angle">
                  <property name="minimumSize">
                   <size>
                    <width>180</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Google Sans</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Cleat Angle</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;For help, please visit&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;# The Spreadsheet Data Converter Github Page&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;# Powered by openSource community!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Google Sans</family>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_load_file">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>&amp;Load Inputs</string>
     </property>
     <addaction name="action_load_fin_plate"/>
     <addaction name="action_load_tension_member"/>
     <addaction name="action_load_bc_end_plate"/>
     <addaction name="action_load_cleat_angle"/>
    </widget>
    <widget class="QMenu" name="menu_open_blank_tab">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>Open Blank Tab</string>
     </property>
     <addaction name="action_blank_fin_plate"/>
     <addaction name="action_blank_tension_member"/>
     <addaction name="action_blank_bc_end_plate"/>
     <addaction name="action_blank_cleat_angle"/>
    </widget>
    <addaction name="menu_load_file"/>
    <addaction name="menu_open_blank_tab"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="action_save_file">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_edit_data">
   <property name="text">
    <string>Edit Data</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_add_data">
   <property name="text">
    <string>&amp;Validate Data</string>
   </property>
  </action>
  <action name="action_column_layout">
   <property name="text">
    <string>&amp;Validate Data</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_column_count">
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_row_count">
   <property name="text">
    <string>32</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_source">
   <property name="text">
    <string>test_data.csv</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_column">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_row">
   <property name="text">
    <string>5</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_toolbar_bottom_selected_cells">
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_text_length">
   <property name="text">
    <string>18</string>
   </property>
  </action>
  <action name="action_save_data_to_file">
   <property name="text">
    <string>&amp;Save Data to file</string>
   </property>
  </action>
  <action name="action_load_fin_plate">
   <property name="text">
    <string>&amp;Fin Plate</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_load_tension_member">
   <property name="text">
    <string>&amp;Tension Member</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_load_bc_end_plate">
   <property name="text">
    <string>&amp;BC End Plate</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_load_cleat_angle">
   <property name="text">
    <string>&amp;Cleat Angle</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_blank_fin_plate">
   <property name="text">
    <string>&amp;Fin Plate</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_blank_tension_member">
   <property name="text">
    <string>&amp;Tension Member</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_blank_bc_end_plate">
   <property name="text">
    <string>&amp;BC End Plate</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_blank_cleat_angle">
   <property name="text">
    <string>&amp;Cleat Angle</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
